config/routes.rb

```diff
@@ -21,6 +21,9 @@
       end
 
       get '/users/saml/metadata' => 'saml#metadata'
+
+      get '/users/lti/redirect', to: 'redirect#redirect'
+      get '/users/lti/do_redirect', to: 'redirect#do_redirect'
```

Comment:
A bit confused about the inconsistency; there's a notion of `LTI` in the route, but not in the name of the controller. I'd suggest going a step further and removing the `user` part as well. Move these methods to the `LtiController`, since they are tied to LTI.

```suggestion
      get '/lti/redirect', to: 'lti#redirect'
      get '/lti/do_redirect', to: 'lti#do_redirect'
```